docs/UserGuide.md

```diff
@@ -173,7 +180,7 @@ Format: `filter buy` or `filter sell`
 * If no clients made any transactions, the transaction section will be blank.
 
 Examples:
-* `filter buy` Displays all buy transactions.
+* `filter buy` displays all buy transactions.
```

Comment:
Not sure if it is better to give the example of `filter sell`

Reply:
will add it in as well